var/spack/repos/builtin/packages/py-dlio-profiler/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDlioProfiler(PythonPackage):
+    """A low-level profiler for capture I/O calls from deep learning applications."""
+
+    homepage = "https://github.com/hariharan-devarajan/dlio-profiler.git"
+    git = "https://github.com/hariharan-devarajan/dlio-profiler.git"
+    maintainers("hariharan-devarajan")
+
+    version("develop", branch="dev")
+    version("master", branch="master")
+    version("0.0.1", tag="v0.0.1")
+    depends_on("cpp-logger@0.0.1")
+    depends_on("brahma@0.0.1")
+    depends_on("gotcha@develop")
+    depends_on("py-setuptools", type="build")
+    depends_on("py-pybind11", type="build")
```

Comment:
Also missing build deps on py-ninja and py-cmake@3.12: (see pyproject.toml)

Reply:
adding missing build deps